src/unix/bsd/apple/b64/mod.rs

```diff
@@ -2,7 +2,6 @@
 
 pub type c_long = i64;
 pub type c_ulong = u64;
-pub type boolean_t = ::c_uint;
 pub type mcontext_t = *mut __darwin_mcontext64;
```

Comment:
I'm not sure about `aarch64-apple-ios` but if it's the same as macOS, we could move this to align.rs on AArch64 (and mod.rs on x86_64).